test/modules/figueroa/DuplicateConfigVars/RandomNumber.chpl

```diff
@@ -1,24 +0,0 @@
-class Random {
```

Comment:
Please mention in the PR message why you are removing this.

Reply:
Done (but I also reshuffled a bit...)